package.json

```diff
@@ -46,6 +49,7 @@
         "punycode": "^2.3.0",
         "release-it": "^17.0.0",
         "rimraf": "^5.0.1",
+        "type-fest": "^4.9.0",
```

Comment:
To ensure it's working as expected, should we move these to "dependencies"?

I'm not very sure what's the best practice for ts-only dependencies. will try it on my local pc later.

Reply:
I usually add them to dependencies 👀

I just want to try and add hints to a few more places first before looking at the exported types :)